test/performance/odp_sched_pktio.c

```diff
@@ -696,13 +758,28 @@ int main(int argc, char *argv[])
 		odp_mb_full();
 	}
 
+	t1 = odp_time_local();
+
 	wait_workers(thread, test_global);
 
+	t2 = odp_time_local();
+
 quit:
 	stop_pktios(test_global);
 	empty_queues();
 	close_pktios(test_global);
 
+	if (test_global->opt.collect_stat) {
+		print_stat(test_global, odp_time_diff_ns(t2, t1));
+
+		/* Encode return value for validation test usage. */
+		if (test_global->rx_pkt_sum > 5000)
```

Comment:
5000 should be in config option or at least in top level define.

Reply:
OK. Added a define in v3.